backend/utils/email.py

```diff
@@ -0,0 +1,33 @@
+from celery import shared_task
+from django.contrib.auth.models import Group
+from django.core.mail import send_mail as django_send_mail
+from django.template.loader import get_template
+
+
+@shared_task(name="utils.send_mail")
+def send_mail(subject, recipient_list, message=None, html_message=None):
+    if recipient_list is None:
+        return False
+    success = django_send_mail(
+        subject=subject,
+        message=message,
+        from_email=None,
```

Comment:
Any reason this is `None`?

Reply:
if we set to None, it uses the default one specified in settings (base.py)